docs/resources/Channel.md

```diff
@@ -668,7 +668,7 @@ Edit a previously sent message. The fields `content`, `embed`, and `flags` can b
 Returns a [message](#DOCS_RESOURCES_CHANNEL/message-object) object. Fires a [Message Update](#DOCS_TOPICS_GATEWAY/message-update) Gateway event.
 
 > info
-> All parameters to this endpoint are optional.
+> All parameters to this endpoint are optional and nullable.
```

Comment:
is there a case where optional != nullable in our api in a PATCH endpoint? I'm trying to understand why this PR is adding "nullable" to everything .. The only thing which may be useful for documentation would be where `null` = `unset` in a PATCH, no?

Reply:
In this PR I've tried to only documented nullability on fields where passing null results in a change (whether that be the value being unset or setting to it's default) rather than the change being ignored (an example of the field being ignored would be patching `owner_id` as `null` for a guild).
Some patch endpoints seem to return an 4xx if you try to pass `null` for certain fields when they cannot be unset (e.g. setting `name` to null when patching a channel returns a 400 with the error message `This field is required`).

Reply:
A good example of the distinction between passing null leading to a change rather than being ignored would be on the `Modify Guild Channel Positions` endpoint where patching with a `position` set to null for a channel leading to it's position being defaulted to 0 (or near if `null` is passed for the position of multiple channels) where as setting the `id` to null leads to that entry just being ignored.
Oddly enough that endpoint has different behaviour to `Modify Guild Role Positions` where leaving out `position` for a role will still lead to it's position being set to the default (unlike `Modify Guild Channel Positions` where the change would be ignored in this situation).